cdx.go

```diff
@@ -0,0 +1,270 @@
+package warc
+
+import (
+	"bufio"
+	"fmt"
+	"mime"
+	"net"
+	"net/http"
+	"net/url"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const (
+	CDXCanonizedURL = 'A' + iota
+	CDXNewsgroup
+	CDXRulespaceCategory
+	CDXCompressedDATOffset
+	_ // 'E'
+	CDXCanonizedFrame
+	CDXLanguageDescription
+	CDXCanonizedHost
+	CDXCanonizedImage
+	CDXCanonizedJumpPoint
+	CDXUnknownFBISChanges // 'K'
+	CDXCanonizedLink
+	CDXMetaTags
+	CDXMassagedURL
+	_ // 'O'
+	CDXCanonizedPath
+	CDXLanguage
+	CDXCanonizedRedirect
+	CDXCompressedSize
+	_ // 'T'
+	CDXUniqness
+	CDXCompressedOffset
+	_ // 'W'
+	CDXCanonizedHrefURL
+	CDXCanonizedSrcURL
+	CDXCanonizedScriptURL // 'Z'
+)
+const (
+	CDXOriginalURL = 'a' + iota
```

Comment:
`cdx.go:46:2: exported const CDXOriginalURL should have comment (or a comment on this block) or be unexported`